hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/checksum/ContainerChecksumTreeManager.java

```diff
@@ -383,6 +373,24 @@ public ByteString getContainerChecksumInfo(KeyValueContainerData data) throws IO
     }
   }
 
+  public static Optional<ContainerProtos.ContainerChecksumInfo> readChecksumInfo(ContainerData data)
```

Comment:
Let's add a comment here similar to what is on `read` indicating that callers don't need to worry about synchronization. This method is now also part of the public API so it should have some javadoc.